src/clib/include/feldspar_c99.h

```diff
@@ -56,6 +56,10 @@
 #include "ivar.h"
 #include "taskpool.h"
 
+#ifndef global
+#define global
```

Comment:
My ifdef comment still remains for this change, but add local inside the ifdef too.